CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
  and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Correct use of messages.
```

Comment:
I wouldn't say "Correct use" but rather "- i18n setup". There was nothing really wrong, we are just moving on to something better.